src/GitHub.UI/Controls/Buttons/GitHubActionLink.cs

```diff
@@ -8,12 +8,21 @@ public partial class GitHubActionLink : Button
         public static readonly DependencyProperty HasDropDownProperty = DependencyProperty.Register(
             "HasDropDown", typeof(bool), typeof(GitHubActionLink));
 
+        public static readonly DependencyProperty TextTrimmingProperty = DependencyProperty.Register(
+            "TextTrimming", typeof(TextTrimming), typeof(GitHubActionLink));
```

Comment:
Small thing, but you can register this property by adding an owner to the `TextBlock.TextTrimmingProperty` like so:

```
public static readonly DependencyProperty TextTrimmingProperty =
    TextBlock.TextTrimmingProperty.AddOwner(typeof(GitHubActionLink));
```

It probably won't make much difference to how the code runs but it tells the reader (at least it tells me) that this is an existing property that is being used, and that I can look up `TextBlock.TextTrimming` for more info.